docs/providers.rst

```diff
@@ -25,6 +25,18 @@ For local development, use the following::
 
 
 
+Each provider could customize `display_name` and `alt_text` properties by respective settings::
+
+    SOCIALACCOUNT_PROVIDERS = {
+        "github": {
+            "display_name": "Login via Github",
+            "alt_text": "Connect your github account."
```

Comment:
For consistency with the other per provider settings, you should probably capitalize `DISPLAY_NAME` and `ALT_TEXT`.